src/applications/vaos/new-appointment/components/ReviewPage/AppointmentDate.jsx

```diff
@@ -25,40 +25,27 @@ export default function AppointmentDate({
         >
           Date and time
         </Heading>
-        {dates?.map(selected => {
-          const dateTime =
-            selected.endsWith('Z') && timezone
-              ? moment(selected).tz(timezone)
-              : moment(selected, 'YYYY-MM-DDTHH:mm:ssZ');
+        {dates?.map((date, index) => {
           return (
-            <>
-              <span>{dateTime.format('dddd, MMMM D, YYYY')}</span>
+            <React.Fragment key={index}>
+              <span>
+                {formatInTimeZone(date, timezone, 'EEEE, MMMM d, yyyy')}
+              </span>
               <br />
               <span>
-                {dateTime.format('h:mm a')} {timezoneAbbr}
+                {formatInTimeZone(date, timezone, 'h:mm aaaa')} {timezoneAbbr}
```

Comment:
Hmmm it looks like moment.js's `a` corresponds to "am/pm" which corresponds to to date-fns's `aaa`. The `aaaa` used here looks slightly different as "a.m./p.m." with periods. Was there any discussion on why this is preferred over the format without periods?